### ISSUE TITLE
Animated View example : how to unblock the view when it has been "opened"

### ISSUE THREAD

**Issue by ghamaide** [react-community/react-native-maps#57]
I'm starting in RN and I've been trying to mess a bit with your Animated example. The example works well but when the red view is brought up, it cannot be closed anymore. How would you work this out ?

![giphy](https://cloud.githubusercontent.com/assets/6986811/12670687/b5d55192-c66a-11e5-9df7-285fcffa2b37.gif)

**Comment by lelandrichardson** [comment]
@ghamaide i'm sorry, but what do you mean by "closed"? the example is built to only pan in the x direction when the red panels are all down.  Also, keep in mind this is just an example and not a "production-ready" idea...

**Comment by mrpatiwi** [comment]
It would be awesome to have that as a separated component

**Comment by cinkster** [comment]
A year later, but I just encountered this in the Animated example myself.

This fix for me was to replace the line

`const topOfMainWindow = ITEM_PREVIEW_HEIGHT + panY.__getValue();`

with

`const topOfMainWindow = ITEM_PREVIEW_HEIGHT - panY.__getValue();`

in both the onMoveShouldSetPanResponder and onStartShouldSetPanResponder methods.

**Comment by euandrei** [comment]
@cinkster 

Thank you so much, do you have any idea how to make it so that we can still scroll content in the view and not overlap with the closing event ?

**Comment by alvelig** [comment]
This seems too old. Trying to clean the repo. Create a new issue if persists
